actgen/train.py

```diff
@@ -246,6 +248,7 @@ def run(self):
             else:
                 s = sp
             utils.every_n_times(self.params['eval_every_n_steps'], step, self.evaluate, step)
+            utils.every_n_times(self.params['eval_every_n_steps'], step, self.save_batch_loss, step, loss)
```

Comment:
If we only eval every N steps, then we only see the loss every N steps, which will be pretty noisy. Can we make a new hyperparameter `save_loss_every_n_steps = 1`?